src/test/java/redis/clients/jedis/commands/jedis/ClientCommandsTest.java

```diff
@@ -233,6 +233,26 @@ public void killUser() {
     }
   }
 
+  @Test
+  public void killMaxAge() throws InterruptedException {
+    long maxAge = 2;
+
+    // sleep twice the maxAge, to be sure
+    Thread.sleep(maxAge * 2 * 1000);
+
+    Jedis client2 = new Jedis(hnp.getHost(), hnp.getPort(), 500);
+    client2.auth("foobared");
+
+    long killedClients = jedis.clientKill(new ClientKillParams().maxAge(maxAge));
+
+    // The reality is that some tests leak clients, so we can't assert
+    // on the exact number of killed clients.
+    assertTrue(killedClients > 0);
+
+    assertDisconnected(client);
+    assertConnected(client2);
```

Comment:
`client2` is still connected. _Ideally_ we should ensure closing this connection as this won't be closed by `@After`.

Reply:
Add a `try` around it. Also in another older test.